src/Android/Accessibility/AccessibilityHelpers.cs

```diff
@@ -115,6 +115,17 @@ public static class AccessibilityHelpers
             "com.ss.squarehome2",
             "com.treydev.pns"
         };
+        
+        // Be sure to keep these entries sorted alphabetically
+        public static Dictionary<string, KnownUsernameField> KnownUsernameFields => new List<KnownUsernameField>
+        {
+            new KnownUsernameField("accounts.google.com","ServiceLogin", "Email"),
+            new KnownUsernameField("amazon.com","signin", "ap_email_login"),
```

Comment:
Given just the authority, how would this work with sub-domains that behave differently, such as `aws.amazon.com` which has a username field prompted first (both iAM and Root accounts), `path="signin", field id="resolving_input"`

Reply:
It has to be added to the list (same approach as `accounts.google.com`, unless I'm misunderstanding the question?):

`new KnownUsernameField("signin.aws.amazon.com","signin", "resolving_input")`

![device-2020-05-06-092205](https://user-images.githubusercontent.com/59324545/81182064-57805900-8f7b-11ea-8167-b27df5e48177.png)

Reply:
Additional clarification - this PR is only for showing the overlay where it wouldn't show before.  The matching algorithms used when the overlay is clicked have not changed.